### DISCUSSION TITLE
rdc.s not showing up in blender

### DISCUSSION THREAD

**Discussion by girardk** [eliemichel/MapsModelsImporter#266] [Q&A]
Hi Elie, I have followed all of the steps using renderdoc 1.25, blender 3.4.1, and mapsmodelsimporter 0.6.2 and the installer loads into blender, and there are no errors popping up when importing a rdc, but no image will load or appear. Not sure what the issue is, please help thank you!

**Comment by tempestblayze** [comment]
I'm having a similar issue. Did a Blender update break this?

**Reply by WingmanMedia** [reply]
How did you even make it work in 3.4? Tried many times to install the add-on and it never happened in 3.4 Almost gave up on it but then found a video on youtube with links in a description to download a zip.  This zip contained a modified Chrome Launcher, RenderDoc, the add-on and executable file to launch Blender3D 3.0

Blender3D 3.0 works with the add-on so you just run 3.0 to use the add-on, bring rdc file in this version, then save a project with it and open a saved blender file in 3.4 or later.

If you have multiple blender versions installed on your PC then you do need that file from the zip, just do everything in 3.0 and then open saved blender project in later versions.

**Comment by tempestblayze** [comment]
Never mind. I should read directions.

"/!\ Do not use the "Download as zip" button of GitHub, make sure you use a release zip instead."

**Reply by girardk** [reply]
So... I feel really dumb here, but what is the difference between a release zip and "download as zip"? I hit the release link (when you click on the word it links to the importer download page) on the repository, and downloaded the first folder there. Am I missing something?

**Reply by eliemichel** [reply]
The `MapsModelImporter-something.zip` file is one I create manually and that is properly set up to be installed as a Blender add on. The Automatic "Source Code" zips are just a raw copy of the repo, which contains more than the addon and where things are laid out differently.
![image](https://github.com/eliemichel/MapsModelsImporter/assets/5802849/d24a27f0-4326-4a35-9e4c-1f2e54763391)

**Comment by germandivision** [comment]
I've experimented with almost every version of renderdoc. they all let me import rdc file. but I couldn't import the mesh file into Blender. My life is shortened from making combinations :)

Should there be a solution to this?

**Comment by Tomas-TDFM** [comment]
Hi. using renderdoc 1.25, blender 3.4.1, and latest MapsModelsImporter 0.6.2 I´m able to load mesh in Blender and everything works. But I´m not able to load mesh from Maps CZ website, there´s always Python error "ZeroDivisionError: float division by zero"... 

I´m capturing map after switching to 3D view and everything looks good in RenderDoc (I can see texture files). 
Is there any difference between Google Maps and Maps CZ capturing? 
Thanks.

EDIT: I found that Outdoor map is working, but Aerial Map not, here is a link with better description: [https://github.com/eliemichel/MapsModelsImporter/issues/264](https://github.com/eliemichel/MapsModelsImporter/issues/264)

**Comment by eliemichel** [comment]
Please all, whenever you fail at importing a specific .rdc file, I need you to **share the file in question** if you want me to try to reproduce and help!

**Reply by f-curado** [reply]
Dear Elie,
I already used Maps Models Importer and Lilly Capture merger successfully before. After months I tied again and I couldn't load rdc in Blender anymore. I tried many different combinations of Blender (4.0, 3.0, 3.4 and 3.6), Maps Models Importer (5.0, 6.0 and 6.2) and Render Doc (1.19 and 1.25). I tried all versions of maps with captures of both Render Doc Versions with all those of Blender and I could not get anything in Blender. 
I get this message in blender:
_AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'inf'_?

I added my captures in 1.19 and 1.25 here:
https://we.tl/t-dlhQXKrJFd

I would be very happy and thankful if you could help me on that. Thanks a lot.

Felipe

**Comment by amirreza144** [comment]
I have a very stupid problem. When I save the .rcd file in my computer from Render Doc, nothing shows up on my computer. I have tried saving in multiple locations, but every time I'd like import the file into the blender, or even when I open the location, I saved the file, nothing is there.
Thanks for your guidance,

**Comment by Bgsketch** [comment]
Hi Elie,

Thank you for this amazing job you've done. I ve been used previous versions of your plug-in + render doc + blender succesfully 3-4 times in the past and it was really helpfull in my work. But now for some reasons i can't import the rdc file into blender. I used Renderdoc 1.31 + Blender 4.2.3 + MapsModelsimporter 0.7.0 + last Chrome version. Do you know what could be the issue? Thanks a lot anyway. Boris 
See my captures (it seems 3d data has been captured correctly) here: https://we.tl/t-MN5mU6JlgZ

**Reply by Bgsketch** [reply]
Hey,

Me again. I found out that it works fine using Microsoft Edge instead of Google for me...Hope it will be helpfull for others...Thanks a lot for this plug in, it's gold!

**Comment by tlpillow2025** [comment]
<img width="960" alt="Screenshot 2025-01-07 182912" src="https://github.com/user-attachments/assets/361b0ca1-2f68-4041-b02d-06e2c00437be" />

I am able to capture the rdc file but when I import into blender nothing shows up. Same results with the test files.

[usbank.zip](https://github.com/user-attachments/files/18339144/usbank.zip)
<img width="703" alt="Screenshot 2025-01-07 182723" src="https://github.com/user-attachments/assets/f1dd98eb-d3c7-47c0-9490-e0470199e3ef" />

**Comment by ghost** [comment]
same problem here using version 0.7.0, Blender 4.1 and RenderDoc 1.31. Tried with both Edge and Chrome. Nothing happens on import into Blender..
